qutebrowser/config/configdata.py

```diff
@@ -1656,4 +1656,9 @@ def data(readonly=False):
     (re.compile(r'^download-remove --all$'), r'download-clear'),
 
     (re.compile(r'^hint links fill "([^"]*)"$'), r'hint links fill \1'),
+
+    (re.compile(r'^set-cmd-text :open -([tb]) {url:pretty}$'),
+                 r'set-cmd-text :open -\1 -i {url:pretty}'),
+    (re.compile(r'^hint links fill :open -t {hint-url}$'),
+                 r'hint links fill :open -t -i {hint-url}'),
```

Comment:
Hmm, I'm not really comfortable about this. Those fixes are mainly for things which are clearly not valid anymore, to automatically fix them in user's configs. Having this here would mean an user can never remove `-i` from those keybindings.... So I'd rather not have this rebinding done automatically.

(I know this sucks... I want to rip out the current config system as soon as the QtWebEngine-stuff is over)